test-resources/swift/cleanup_rules/input/SampleClass.swift

```diff
@@ -19,5 +19,6 @@ class SampleClass {
         isEnabled = v2 && TestEnum.stale_flag_one.isEnabled 
         isEnabled = v2 && (TestEnum.stale_flag_one.isEnabled && true)
         isEnabled = (TestEnum.stale_flag_one.isEnabled && true) && v2
+        isEnabled = !TestEnum.stale_flag_one.isEnabled
```

Comment:
have u added a rule for simplifying `!` ?

Reply:
Yes actually, I have added a rule to detect `!TestEnum.stale_flag.isEnabled` and replace with `treated_complement` which is supplied as `false`

Reply:
how about adding rules that converts `!true` -> `false` and `!false` -> `true`. 
Look here - https://github.com/uber/piranha/blob/master/src/cleanup_rules/go/rules.toml#L23

Reply:
I will add them, but that wouldn't work with `!TestEnum.stale_flag.isEnabled` as `!` is associated with `TestEnum` rather than whole expression.

Reply:
I see. This is a bug. Do you want to report this to upstream?